dotnet/Microsoft.iOS.Sdk/data/UnixFilePermissions.xml

```diff
@@ -5,4 +5,7 @@
 	<!-- mlaunch -->
 	<File Path="tools/bin/mlaunch" Permission="755" />
 	<File Path="tools/lib/mlaunch/mlaunch.app/Contents/MacOS/mlaunch" Permission="755" />
+     <!-- xcsync -->
```

Comment:
Do we only use xcsync for iOS? Or do we need it for tvOS, Mac Catalyst and/or macOS as well?

Reply:
All four

Reply:
Then you need to add these changes to the other three UnixFilePermission.xml files as well.